tests/e2e/taqueria-plugin-ipfs-pinata.spec.ts

```diff
@@ -1,132 +1,186 @@
 
-	// Skipping due to help output changing
-	test.skip('ipfs pinata plugin should show the correct help contents', async () => {
-		const publishRun = await exec(`taq --help`, { cwd: taqueriaProjectPath });
-		expect(publishRun.stdout).toBe(contents.helpContentsIPFSPinataPlugin);
-	});
+describe('IPFS Pinata plugin E2E tests for Taqueria CLI', () => {
+	test('publish will offer contextual help', async () => {
+		const { execute, cleanup, exists } = await prepareEnvironment();
+		const {} = await execute('taq', 'init test-project');
+		await exists('./test-project/.taq/config.json');
+		const {} = await execute('taq', 'install ../taqueria-plugin-ipfs-pinata', './test-project');
+		await exists('./test-project/node_modules/@taqueria/plugin-ipfs-pinata/index.js');
 
-	// Skipping due to help output changing
-	test.skip('ipfs pinata plugin should show the correct help contents for publish', async () => {
-		const publishRun = await exec(`taq publish --help`, { cwd: taqueriaProjectPath });
-		expect(publishRun.stdout).toBe(contents.helpContentsIPFSPinataPluginPublish);
-	});
+		const { stdout: stdout2 } = await execute('taq', 'publish --help', './test-project');
+		expect(stdout2).toEqual(expect.arrayContaining(['Upload and pin files using your pinata account.']));
 
-	// Skipping due to help output changing
-	test.skip('ipfs pinata plugin should show the correct help contents for pin', async () => {
-		const publishRun = await exec(`taq pin --help`, { cwd: taqueriaProjectPath });
-		expect(publishRun.stdout).toBe(contents.helpContentsIPFSPinataPluginPin);
+		await cleanup();
 	});
 
-	test('ipfs pinata plugin should warn for publish if no env jwt token exists', async () => {
-		const publishRun = await exec(`taq publish ./ipfs/0001.txt`, { cwd: taqueriaProjectPath });
-		expect(publishRun.stderr).toMatch(/jwt *token.*not *found/i);
+	test('pin will offer contextual help', async () => {
+		const { execute, cleanup, exists } = await prepareEnvironment();
+		const {} = await execute('taq', 'init test-project');
+		await exists('./test-project/.taq/config.json');
+		const {} = await execute('taq', 'install ../taqueria-plugin-ipfs-pinata', './test-project');
+		await exists('./test-project/node_modules/@taqueria/plugin-ipfs-pinata/index.js');
+
+		const { stdout: stdout2 } = await execute('taq', 'pin --help', './test-project');
+		expect(stdout2).toEqual(expect.arrayContaining(['Pin a file already on ipfs with your pinata account.']));
+
+		await cleanup();
 	});
 
-	// Skipping this test until issue https://github.com/ecadlabs/taqueria/issues/1188 is resolved
-	test.skip('ipfs pinata plugin should warn for pin if no env jwt token exists', async () => {
-		const publishRun = await exec(`taq pin QmZcEXgfE9K4zFVkTrqW5x4skvEqpNajZzvb7frrWnkQa5`, {
-			cwd: taqueriaProjectPath,
-		});
-		expect(publishRun.stderr).toMatch(/jwt *token.*not *found/i);
+	test('publish will error if no env jwt token exists', async () => {
```

Comment:
"publish *throws* an error if ..."

Always the active voice in tests

Reply:
https://www.nasa.gov/seh/appendix-c-how-to-write-a-good-requirement